exporters/metric/prometheus/prometheus.go

```diff
@@ -173,6 +162,14 @@ func (e *Exporter) Controller() *pull.Controller {
 	return e.controller
 }
 
+func (e *Exporter) ExportKindFor(*metric.Descriptor, aggregation.Kind) export.ExportKind {
+	// NOTE: Summary values should use Delta aggregation, then be
+	// combined into a sliding window, see the TODO below.
```

Comment:
Thanks. I downgraded the TODO on the following line to a NOTE because I don't have enough information (and the public documents on GaugeDelta are sketchy at best).